presto-parser/src/main/java/com/facebook/presto/sql/parser/AstBuilder.java

```diff
@@ -1164,6 +1164,15 @@ public Node visitLogicalBinary(SqlBaseParser.LogicalBinaryContext context)
         Expression left = (Expression) visit(context.left);
         Expression right = (Expression) visit(context.right);
 
+        if (operator.equals(LogicalBinaryExpression.Operator.OR) &&
+                left instanceof LikePredicate && right instanceof LikePredicate) {
+            if (((LikePredicate) left).getValue().equals(((LikePredicate) right).getValue())) {
+                warningConsumer.accept(new ParsingWarning(
+                        "Logical OR for the two LIKE operators is being performed. " +
```

Comment:
I find the warning not very easy to understand. What about something more instructive like `Multiple "... LIKE" is not efficient, please use REGEXP_LIKE instead`.